modules/dnn/src/layers/convolution_layer.cpp

```diff
@@ -253,11 +255,13 @@ class ConvolutionLayerImpl CV_FINAL : public BaseConvolutionLayerImpl
 {
 public:
     enum { VEC_ALIGN = 8, DFT_TYPE = CV_32F };
-    Mat weightsMat;
+    Mat weightsMat, fastWeights;
```

Comment:
It is better to put this near `fastConv2dImpl`

Also it makes sense to add documetation/information about its layout and the difference from `weighstMat`.

Reply:
Done.